### PR TITLE
Something amuck with _form_select

### PR DESCRIPTION
**Pull request by mlworle**
Error Below: @courseload was instantiated in courseloads_controller.rb.  Hmm. Rake routes doesn't show course_loads_path. though I changed and corrected the paths  to courseloads. What did I miss? 

NoMethodError in Courseloads#new

Showing /home/spock/rails/Attinderance/app/views/courseloads/_form_select.html.erb where line #1 raised:

undefined method `course_loads_path' for #<#<Class:0x007fc3f13d3f08>:0x007fc3f27f4708>

Extracted source (around line #1)

1 <%= form_for(@courseload) do |f|  %>
  <%= f.select(teacher.name, teacher.id, Teacher.all.collect {|p| [p.name, p.id]}) %>
  <%= f.select(course.name, course.id, Course.all.collect {|p| [p.name, p.id]}) %>

  <%= f.submit  %>
<% end  %>

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -7,4 +7,5 @@
   resources :students
   resources :teachers
   resources :courses 
+  resources :courseloads
```
Comment (edwardloveall) [inline config/routes.rb]: This likely needs to be `:course_loads` with the underscore. When you have a constant with multiple words (e.g. `CourseLoad`) it separates each word by an underscore. You can see this by typing `CourseLoad.to_s.underscore` in the console.

**Comment by mlworle** [comment]
Yep, sorry. I knew that...  
Now its an issue with the courseload_params


ActionController::ParameterMissing in CourseLoadsController#create
param is missing or the value is empty: teacher
Extracted source (around line #19):

  def courseload_params
    params.require( :teacher).permit(:name, :id)
    params.require( :course).permit( :name, :id)
  end

**Comment by mlworle** [comment]
Looks like it was the parameters in create action. Checked it in the console and the record is there. Would like to close this branch and work on the views in another branch.

**Comment by edwardloveall** [comment]
Do you have time today for a session? This is a really good opportunity to talk about debugging. I don't think you should close this, I think we should keep working on it until it works.

**Comment by mlworle** [comment]
I have until noon. Would that work?

**Comment by edwardloveall** [comment]
Yeah I have until a little after 11. I'll hop on screen sharing and you can message me when you're ready.

**Comment by mlworle** [comment]
Working. Clunky and very slow!

**Comment by edwardloveall** [comment]
Yeah looks good! :shipit:

**Pull request merged**